src/sgame/sg_cmds.cpp

```diff
@@ -4359,6 +4359,7 @@ static const commands_t cmds[] =
 	{ "noclip",          CMD_CHEAT_TEAM,                      Cmd_Noclip_f           },
 	{ "notarget",        CMD_CHEAT | CMD_TEAM | CMD_ALIVE,    Cmd_Notarget_f         },
 	{ "pubkey_identify", CMD_INTERMISSION,                    Cmd_Pubkey_Identify_f  },
+	{ "putcon",          CMD_CHEAT,                           Cmd_PutCon_f           },
```

Comment:
I somehow managed to add the end of a connection while being dead. Let's not allow that:
```suggestion
	{ "putcon",          CMD_CHEAT | CMD_TEAM | CMD_ALIVE,    Cmd_PutCon_f           },
```

Reply:
I don' t agree that putcon must require not being spectator, as the connection start or end will just end under the feet. Well, I have not tested that, but that's the behavior I'd expect, and it makes it easier sometimes.
The alive check, though, makes sense, but is a spectator alive?
There are other problems in there, like, you can't really do some navcon commands while following, for some reason. That's annoying enough, but I don't know how to change that, yet.

Reply:
If you are a spectator; how are you supposed to get the closest point on your navmesh? GetFeetPoint uses that.

I'm not sure it would be worthwhile to do the testing and integration, even if it worked.

Reply:
no, GetFeetPoint will find the navmesh position which is the closest to the entity's feet.  This is different, and there's even a message saying if things worked as expected (and now that I think about it, there's none when it does not?)

Reply:
I'm not sure how that's supposed to work. What's the spectator's navmesh? Unfortunately you currently have no feedback so it's hard to know if it worked

Reply:
you don't edit the spectator's navmesh, you edit the navmesh you selected before.
There is feedback about if the putcon worked, but none for addcon, that I kept pristine.

Reply:
I had no feedback at all with putcon when I tried. Is it because I didn't select a class with `navedit`? Sounds like something worth fixing anyhow

Reply:
the feedback is a Log::Notice which appear on success. See https://github.com/Unvanquished/Unvanquished/pull/1519/commits/b3f078bacd9529df7144728ad331640430a30ad4#diff-385d371b213d00191fa775c5ba68d4edd1bb55cfdfd80e3e1da2f7d3142cb4b4R453

Reply:
In my case, I get no feedback at all

![Screenshot from 2021-10-03 22-35-14](https://user-images.githubusercontent.com/59283660/135770707-568074b8-334f-423b-9e8a-bdd68bce9f4c.png)

Step to reproduce:
* join a map, don't join a team
* `/putcon start oneway`
* `/putcon end`

Reply:
Will investigate.